src/layout/Dashboard.tsx

```diff
@@ -33,7 +29,6 @@ const Dashboard = (props: Props): ReactElement => {
       <Container className={classes.content}>
         <ErrorBoundary>
           <>
-            <AlertVersion />
```

Comment:
I asked for removing this for the internal testing. It would be good to improve this, because the current version is very annoying (it comes back after a reload or when you open the page from the menu) and it requires us to have exactly the same version of Bee bundled with the desktop as the version that the bee-js supports that is bundled with the dashboard.

Reply:
OK we can remove it in this PR, but we can definitely reintroduce it with better mechanism. `bee-js`has a different method to check version as that is less strict. And we can also remember the dismissal of the warning with specific version and timestamp to show it when the version changes or when e.g. 7 days pass.